controllers/users.js

```diff
@@ -457,6 +458,7 @@ const updateSelf = async (req, res) => {
   try {
     const { id: userId, roles: userRoles, discordId } = req.userData;
     const devFeatureFlag = req.query.dev === "true";
+    const devFlag = req.query.dev;
```

Comment:
Hey, so don't we already have a dev=true flag, for something else?

How are you making sure that your flag is not triggering the other code?

Reply:
I changed the logic to boolean, hence using the same flag and not making a new one.